Formula/ebmc.rb@5.0

```diff
@@ -0,0 +1,23 @@
+class Ebmc < Formula
+  desc "Model Checker for SystemVerilog"
+  homepage "https://www.cprover.org/ebmc/"
+  url "https://github.com/diffblue/hw-cbmc.git",
+    revision: "f2f3a40951f9e505a90621b430035b811dfc2b1a"
+  version "5.0"
+  license "BSD-3-Clause"
+
+  uses_from_macos "flex" => :build
+  uses_from_macos "curl" => :build
+  depends_on "bison" => :build
+
+  def install
+    system "make", "-C", "lib/cbmc/src", "minisat2-download"
+    system "make", "-C", "src", "-j4"
```

Comment:
Maybe use `-j` instead of hard-coding 4 parallel processes?

Reply:
Homebrew apparently sets MAKEFLAGS accordingly.